hardhat.config.js

```diff
@@ -15,6 +15,18 @@ task('accounts', 'Prints the list of accounts', async (taskArgs, hre) => {
 // You need to export an object to set up your config
 // Go to https://hardhat.org/config/ to learn more
 
+const rinkebyGatewayUrl = process.env.NODE_ENV === 'test'
+  ? 'https://rinkeby.infura.io/v3/token'
+  : process.env.RINKEBY_GATEWAY
+
+const mainnetGatewayUrl = process.env.NODE_ENV === 'test'
+  ? 'https://mainnet.infura.io/v3/token'
+  : process.env.MAINNET_GATEWAY
+
+const deployerPrivateKey = process.env.NODE_ENV === 'test'
+  ? '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff90'
```

Comment:
do we want to define a constant holding this to understand its meaning!

Reply:
This is really just a bogus key so that hardhat won’t bark before running tests... I could just add a comment.